src/components/Pills/Pill.test.tsx

```diff
@@ -46,6 +46,20 @@ describe('Pill', () => {
     expect(container).toMatchSnapshot();
   });
 
+  test('Pill should render with icon to the right of its label', () => {
+    const { container } = render(
+      <Pill
+        alignIcon={PillIconAlign.Right}
```

Comment:
would you consider using to use Start/End like flexbox so this is not so confusing in RTL scenarios?

Reply:
Good point. Perhaps this might be something to normalize in Octuple v3?